deploy/helm/cf-operator/values.yaml

```diff
@@ -7,7 +7,11 @@ image:
 
 operator:
   watchNamespace: ""
-  webhookUseServiceReference: true
+  webhook:
```

Comment:
We should use webhookUseServiceReference by default as recommended by Kubernetes docs [0]. what do you think ?

[0] https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#service-reference

Reply:
done